ts/src/okx.ts

```diff
@@ -842,6 +842,7 @@ export default class okx extends Exchange {
         params = this.omit (params, 'instType');
         const type = this.safeString (params, 'type');
         if ((type === undefined) && (instType !== undefined)) {
+            params = this.extend (params);
```

Comment:
No need to extend it here, cause we already overrided it in line 842.

Reply:
oh, i've not noticed it, right.